### PR TITLE
fix(DB/creature): Add random movement to critters in "Pit of Saron"

### PR DESCRIPTION
**Pull request by Stoabrogga**
##### CHANGES PROPOSED:
Add random movement to the rats and spiders in "Pit of Saron". As it is now, they don't move.

##### ISSUES ADDRESSED:
none

##### TESTS PERFORMED:
tested successfully in-game

##### HOW TO TEST THE CHANGES:
- ```.go 444.03  210.355 528.708 658```
- there should be a few rats and spiders near the entrance

##### KNOWN ISSUES AND TODO LIST:
none

##### Target branch(es):
Master
 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Comment by pklloveyou** [comment]
image
image
Success, good work

**Comment by FrancescoBorzi** [comment]
excellent @Stoabrogga and @pklloveyou !

File: data/sql/updates/pending_db_world/rev_1565620827529880771.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1565620827529880771');
+
+UPDATE `creature` SET `MovementType` = 1, `spawndist` = 2 WHERE `map` = 658 AND `id` IN (2110,14881);
```
Comment (poszer) [inline data/sql/updates/pending_db_world/rev_1565620827529880771.sql]: This is not correct.

How i can see creatures with 2110 & 14881 are Stranglethorn Tiger & Torn Fin Muckdweller located in Hillsbrad Foothills and Stranglethorn Vale not the npc's from Pit of Saron how you mentioned it. 

In this statement you're updating creatures, not creature_template (Rat creature_template is 2110 and Spider creature_template is 14881).

And also while you testing @pklloveyou , test it properly before commenting.

**Comment by poszer** [comment]
How i know

> excellent @Stoabrogga and @pklloveyou !

And also don't approve code when it's not tested , right ?

**Pull request merged**